packages/reporter/webpack.config.ts

```diff
@@ -0,0 +1,130 @@
+import webpack, { Configuration, optimize, ResolvePlugin } from 'webpack'
```

Comment:
Could we somehow abstract this configuration and the shared dependencies between the reporter and runner to make it easier to edit and upgrade the dependencies in the future?

Reply:
that is probably a good idea

Reply:
ok, how do you like this solution https://github.com/cypress-io/cypress/pull/4103/commits/cafedfae7c00370e81a4e24e767465cafea43265

Reply:
Much better! 👍